dir.targets

```diff
@@ -55,8 +56,13 @@
       <RuntimePath>$(RuntimePath)</RuntimePath>
     </BinPlaceConfiguration>
 
+    <BinPlaceConfiguration Condition="'$(IsNETCoreApp)' == 'true' AND '$(IsTestProject)' == 'true' AND '$(BinPlaceNETCoreAppTests)' == 'true'" Include="netcoreapp-$(_bc_OSGroup)">
+      <PackageFileRuntimePath>$(NETCoreAppPackageRuntimePath)\..\tests</PackageFileRuntimePath>
+      <RuntimePath>$(NETCoreAppPackageRuntimePath)\..\tests</RuntimePath>
+    </BinPlaceConfiguration>
+
     <!-- binplace to directories for packages -->
-    <BinPlaceConfiguration Condition="'$(IsNETCoreApp)' == 'true' AND '$(BuildingNETCoreAppVertical)' == 'true'" Include="netcoreapp-$(_bc_OSGroup)">
+    <BinPlaceConfiguration Condition="'$(IsNETCoreApp)' == 'true' AND '$(IsTestProject)' != 'true' AND '$(BuildingNETCoreAppVertical)' == 'true'" Include="netcoreapp-$(_bc_OSGroup)">
```

Comment:
Yes, bin/pkg/netcoreapp/runtime will contain test libraries without this additional condition.